internal/controlplane/handlers_providers_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/google/uuid"
```

Comment:
I decided to reuse these test cases to test the deletion codepath in `GitHubProviderClassManager`